outputs.tf

```diff
@@ -60,182 +60,207 @@ output "vpc_main_route_table_id" {
 
 output "vpc_secondary_cidr_blocks" {
   description = "List of secondary CIDR blocks of the VPC"
-  value       = ["${aws_vpc_ipv4_cidr_block_association.this.*.cidr_block}"]
+  value       = [aws_vpc_ipv4_cidr_block_association.this.*.cidr_block]
 }
 
 output "private_subnets" {
   description = "List of IDs of private subnets"
-  value       = ["${aws_subnet.private.*.id}"]
+  value       = [aws_subnet.private.*.id]
```

Comment:
I am not going to comment on every line, but these square brackets are not needed.